### PR TITLE
refactor(tiktok): patch name convention.

### PR DESCRIPTION
**Pull request by d4rkk3y**

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/tiktok/ad/patch/TiktokAdsPatch.kt [hunk 1]
```diff
@@ -19,7 +19,7 @@ import org.jf.dexlib2.iface.instruction.TwoRegisterInstruction
 import org.jf.dexlib2.iface.reference.FieldReference
 
 @Patch
-@Name("tiktok-ads")
+@Name("ads")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/tiktok/ad/patch/TiktokAdsPatch.kt:22]:
```suggestion
@Name("hide-ads")
```

File: src/main/kotlin/app/revanced/patches/tiktok/misc/forcelogin/patch/DisableForceLoginPatch.kt [hunk 2]
```diff
@@ -14,7 +14,7 @@ import app.revanced.patches.tiktok.misc.forcelogin.fingerprints.MandatoryLoginSe
 import app.revanced.patches.tiktok.misc.forcelogin.fingerprints.MandatoryLoginServiceFingerprint2
 
 @Patch
-@Name("tiktok-force-login")
+@Name("mandatory-login")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/tiktok/misc/forcelogin/patch/DisableForceLoginPatch.kt:17]:
```suggestion
@Name("disable-login-requirement")
```

File: src/main/kotlin/app/revanced/patches/tiktok/misc/loginfallback/patch/TikTokLoginFallbackPatch.kt [hunk 3]
```diff
@@ -14,7 +14,7 @@ import app.revanced.patches.tiktok.misc.loginfallback.fingerprints.GoogleAuthAva
 import app.revanced.patches.tiktok.misc.loginfallback.fingerprints.GoogleOneTapAuthAvailableFingerprint
 
 @Patch
-@Name("tiktok-web-login")
+@Name("web-login")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/tiktok/misc/loginfallback/patch/TikTokLoginFallbackPatch.kt:17]:
```suggestion
@Name("fix-google-login")
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
Looks good, i've left some suggestions from me on the naming. Before merge the changes should also be reflected to the class names, such as omitting "Tiktok", because the patch is already in the correct package.

**Review by oSumAtrIX — APPROVED** [review]
Did you also consider the integrations & package names?

**Comment by d4rkk3y** [comment]
> Did you also consider the integrations & package names?

What should I change in integrations?

**Comment by oSumAtrIX** [comment]
No i was just wondering, if there were any changes required for the integrations as well, checked and it seems fine.

**Pull request merged**